src/main/scala/io/flow/countries/AvailableCountriesProvider.scala

```diff
@@ -1,11 +1,12 @@
-package io.flow.localization
```

Comment:
How about we put everything in `io.flow.localization`?

- `io.flow.localization.country`
- `io.flow.localization.rate`
- `io.flow.localization.pricing`